Makefile

```diff
@@ -27,10 +28,10 @@ all: kubeapps/dashboard kubeapps/apprepository-controller kubeapps/kubeops kubea
 # Currently the go projects include the whole repository as the docker context
 # only because the shared pkg/ directories?
 kubeapps/%:
-	DOCKER_BUILDKIT=1 docker build -t kubeapps/$*$(IMG_MODIFIER):$(IMAGE_TAG) --build-arg "VERSION=${VERSION}" -f cmd/$*/Dockerfile .
+	DOCKER_BUILDKIT=1 docker build --platform "linux/$(TARGET_ARCHITECTURE)" -t kubeapps/$*$(IMG_MODIFIER):$(IMAGE_TAG) --build-arg "VERSION=${VERSION}" -f cmd/$*/Dockerfile .
```

Comment:
I'd do `--platform "$(TARGET_ARCHITECTURE)"` instead. If Windows containers are ever supported in buildkit, we'd like to also have them.

Besides, I'd rather add some notes commenting the following:

The target platforms supported by buildkit are `linux/amd64, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6`

However, if a user happens to use Windows Containers (not WSL or VM), it won't work as buildkit doesn't support Windows (see https://github.com/microsoft/Windows-Containers/issues/34)

Note that it refers to the container architecture. For instance, a Docker for Windows or Docker for Mac usually runs Linux containers,so we only need to build these two target platforms: `linux/amd64` and `linux/arm64`

Reply:
After discussing this topic, we've agreed on just providing the `TARGET_ARCHITECTURE` and keeping hardcoded the operating system, as we won't likely support Windows containers in the near future and have to provide the whole platform string (eg. `linux/arm64`) would add more complexity. 

In addition, instead of adding a not regarding the supported platforms, I've decided to add a validation in the code, so `Make` will fail with a descriptive error if an unsupported architecture is provided.